.github/workflows/test_fuzzer_benchmarks.py

```diff
@@ -100,17 +105,30 @@ def do_build(build_type, fuzzer):
     else:
         raise Exception('Invalid build_type: %s' % build_type)
 
+    if always_build:
+        return make_builds(benchmarks, fuzzer)
+
+    changed_files = diff_utils.get_changed_files()
+    changed_fuzzers = change_utils.get_changed_fuzzers(changed_files)
+    if fuzzer in changed_fuzzers:
+        return make_builds(benchmarks, fuzzer)
+
+    changed_benchmarks = set(change_utils.get_changed_benchmarks(changed_files))
+    benchmarks = benchmarks.intersection(changed_benchmarks)
     return make_builds(benchmarks, fuzzer)
 
 
 def main():
     """Build OSS-Fuzz or standard benchmarks with a fuzzer."""
-    if len(sys.argv) != 3:
-        print('Usage: %s <build_type> <fuzzer>' % sys.argv[0])
+    if len(sys.argv) != 4:
+        print('Usage: %s <build_type> <fuzzer> <always_build_fuzzer>' %
```

Comment:
Lets just put always_build_fuzzer as a global in this file, it looks weird to put in this file argument. also, someone can manually run this script as well.

Reply:
Done. but if someone were running manually, I think the global is also weird since script will inexplicably behave differently for AFL.